src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+    public static final int MAX_NUMBER_FIGURES = 6;
+    public static final int MAX_NUMBER_SIDE = 20;
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure(String randomColor) {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
```
randomize color in the method